site/themes/carvel/layouts/partials/_banner.html

```diff
@@ -1,5 +1,5 @@
-<!-- <div class="pageinfo pageinfo--red" style="text-align:center; margin:1px; display:flex; justify-content: center">
+<div class="pageinfo pageinfo--red" style="text-align:center; margin:1px; display:flex; justify-content: center">
   <span style="margin: 0px; padding: 8px; background-color:#fff7b3; width: 100%;">
-    Come find us at KubeCon + CloudNativeCon Europe! Keep an eye out on Twitter <a href="https://twitter.com/carvel_dev" target="_blank">@carvel_dev</a> and Slack <a href="https://kubernetes.slack.com/archives/CH8KCCKA5" target="_blank">#carvel</a> for further details.
+    Introducing kctrl Package authoring commands. Find out how they make Package authoring simpler <a href="/blog/kctrl-pkg-authoring-cmds/">here</a>!
```

Comment:
@microwavables ,@renuy 
Any suggestions for banner text to make more prominent

Reply:
Introducing kctrl Package authoring commands. Read all about it here

@microwavables ?